sormas-backend/src/test/java/de/symeda/sormas/backend/performance/PerformanceLogAnalysisGenerator.java

```diff
@@ -85,7 +85,7 @@ public class PerformanceLogAnalysisGenerator {
 	public static final int TIME_TRESHOLD_ORANGE = 300;
 	public static final int TIME_TRESHOLD_RED = 1000;
 
-	public static final String OUTPUT_DIRECTORY = "target/performanceLogAnalysis/";
+	public static String OUTPUT_DIRECTORY = "target/performanceLogAnalysis/";
```

Comment:
Uppercase violates convention. Make this an instance variable

Reply:
Done.